apps/www/README.md

```
@@ -2,35 +1,55 @@
+   ```bash
+   pnpm prisma migrate dev --name init
+   ```
+
+5. **Testing the Setup**:
+- You can test your setup by running the development server:
+   ```bash
+   pnpm dev
+   ```
+   - Access your application at [Login](http://localhost:3003/auth).
+
+    - Access your application at [Upastash](http://localhost:3001/api/health).
```

Markdownlint (0.35.0)

Expected: 3; Actual: 4
Inconsistent indentation for list items at the same level

(MD005, list-indent)